main.py

```diff
@@ -104,6 +118,8 @@
         reaction_event_repository,
     )
 
+    slack_view = SlackViewV1(auth_controller, config.slack_client_id, config.slack_scopes, config.admin_emails)

```

Comment:
Refactor config to a model or typed dict, you can look at GoogleCredentials or AWSConfig

Reply:
Sure i will take a look

Reply:
Agreed on this. Often people start to use a proper struct sometimes just to define the dependencies of a service as it's easier to reason about